packages/shared/lib/contexts/governance/utils/isVotingForProposal.ts

```diff
@@ -1,22 +1,13 @@
 import { get } from 'svelte/store'
-
-import type { ParticipationOverview, TrackedParticipationOverview } from '@iota/wallet'
-
-import { participationOverview } from '@contexts/governance/stores'
-import { getParticipationOverview, selectedAccountIndex } from '@core/account'
+import type { TrackedParticipationOverview } from '@iota/wallet'
+import { selectedAccountIndex } from '@core/account'
+import { getParticipationsForProposal } from './getParticipationsForProposal'
 
 export async function isVotingForProposal(
     proposalId: string,
     accountIndex = get(selectedAccountIndex)
 ): Promise<boolean> {
-    let overview: ParticipationOverview
-    if (accountIndex === get(selectedAccountIndex)) {
-        overview = get(participationOverview)
-    } else {
-        overview = await getParticipationOverview(accountIndex)
-    }
```

Comment:
Why did you remove this? It was an optimization to rely on the `participationOverview`, since that's a slow wallet API call.

Reply:
i moved it into another function `getParticipationsForProposal.ts` because i needed that part somewhere else